proposals/release-approval-process.md

```diff
@@ -0,0 +1,38 @@
+
+**Rejecting a release:** A project maintainer MAY choose to reply with REJECT. A project maintainer posting a REJECT MUST include a list of concerns or links to written documentation for those concerns (e.g. GitHub issues or mailing-list threads). The project maintainers SHOULD try to resolve the concerns and wait for the rejecting maintainer to change their opinion to LGTM. However, a release MAY pass with REJECTs, as outlined in the previous paragraph.
+
+## Security fixes
+
+Security fix releases MUST use security@opencontainers.org instead of dev@opencontainers.org, but should otherwise follow the standard [list-based voting process](#list-based-voting). The security@opencontainers.org email includes all members of the TOB; the TOB will guide the security sensitive release with project maintainers.
+
+## Parallel proposals
+
+A single repository MAY have several release proposals in parallel. However each proposed release after the first MUST be based on a previous release that has already landed.
+
+For example, runtime-spec maintainers may propose a v1.0.0-rc2 on the 1st of the month and a v0.9.1 bugfix on the 2nd of the month. They may not propose a v1.0.0-rc3 until the v1.0.0-rc2 is accepted (on the 7th if the vote initiated on the 1st passes).
+
+## Specifications
+
+The OCI maintains three categories of projects: specifications, applications, and conformance-testing tools. However, specification releases have special restrictions in the [OCI charter][charter]:
+
+* They are the target of backwards compatibility (§7.g), and
+* They are subject to the OFWa patent grant (§8.d and e).
+
+To avoid unfortunate side effects (onerous backwards compatibity requirements or Member resignations), the following additional procedures apply to specification releases:
+
+**Planning a release:** Every OCI specification project SHOULD hold meetings that involves maintainers reviewing pull requests, debating outstanding issues, and planning releases. This meeting MUST be advertised on the project README and MAY happen on a phone call, video conference, or on IRC. Maintainers MUST send updates to the dev@opencontainers.org with results of these meetings. Before the specification reaches v1.0.0, the meetings SHOULD be weekly.  Once a specification has reached v1.0.0, the maintainers may alter the cadence, but the meeting cadence MUST NOT be greater than once every four weeks. The release plans, corresponding milestones and estimated due dates MUST be published on GitHub (e.g. https://github.com/opencontainers/runtime-spec/milestones). GitHub milestones and issues are only used for community organization and all releases MUST follow the [list-based voting process](#list-based-voting).
```

Comment:
On Mon, Jun 27, 2016 at 05:44:09PM -0700, Vincent Batts wrote:
> > +**Planning a release:** Every OCI specification project SHOULD
> >  hold meetings that involves maintainers reviewing pull requests,
> >  debating outstanding issues, and planning releases… Before the
> >  specification reaches v1.0.0, the meetings SHOULD be weekly.
> >  Once a specification has reached v1.0.0, the maintainers may
> >  alter the cadence, but the meeting cadence MUST NOT be greater
> >  than once every four weeks…
>
> Is this saying that immediately after releasing v1.0.0 that the
> weekly call will go to once a month?

What gives you that impression?  I read it as:

a. Maintainers don't have to hold meetings ever if they don't want
   (initial SHOULD in my excerpt).
b. Before 1.0, this document recommends spec-project meetings weekly.
c. After 1.0, this document does not have a recommended meeting
   frequency.
d. After 1.0, this document sets a maximum meeting frequency of 28
   days.

So after 1.0, the project maintainers may opt to transition to four
weeks, but they don't have to.

I agree that the section could be reworded for increased clarity
though.  Having a MUST limit on frequency (that only applies after
1.0?) is strange if having meetings at all is only a SHOULD.  I'll
file a PR against @philips branch (if he doesn't update it on his own)
once I see how philips/tob#2 works out.